69.md

```diff
@@ -0,0 +1,126 @@
+* MAY specify a `value_maximum` satoshi value for zapped votes to be included in the tally
+* MAY specify a `value_minimum` satoshi value for zapped votes to be included in the tally
+  * `value_minimum` MUST be less than or equal to `value_maximum`, when both are specified
+* MAY include a `consensus_threshold` (0-100), representing a percentage threshold for any single vote option to achieve poll consensus
+  * a `consensus_threshold` of '0' indicates no threshold is specified.
+
+```json
+{
+  "id": <32-bytes lowercase hex-encoded sha256 of the serialized event data>
+  "pubkey": <32-bytes lowercase hex-encoded public key of the event creator>,
+  "created_at": <unix timestamp in seconds>,
+  "kind": 6969,
+  "tags": [
+    ["e", <32-bytes hex of the id of the poll event>, <primary poll host relay URL>],
+    ["p", <32-bytes hex of the key>, <primary poll host relay URL>],
+    ["poll_option", "0", "poll option 0 description string"],
+    ["poll_option", "1", "poll option 1 description string"],
+    ["poll_option", "n", "poll option <n> description string"],
+    ["value_maximum", "maximum satoshi value for inclusion in tally"],
+    ["value_minimum", "minimum satoshi value for inclusion in tally"],
+    ["consensus_threshold", "required percentage to attain consensus <0..100>"],
+    ["closed_at", "unix timestamp in seconds"],
+  ],
+  "ots": <base64-encoded OTS file data>
+  "content": <primary poll description string>,
+  "sig": <64-bytes hex of the signature of the sha256 hash of the serialized event data, which is the same as the "id" field>
+}
+```
+
+## Zap voting
+
+Poll options are voted on by sending [zap events](57.md) (referencing the original poll event's `e` and `p` values) that indicate their chosen vote option in a `poll_option` tag. To ensure all eligible votes are included in the tally, all `e` and `p` tags must specify the same primary hosting relay.
+
+A voting client:
+* SHOULD NOT allow submission of zap events with amounts greater than `value_maximum` (when specified)
+* SHOULD NOT allow submission of zap events with amounts less than `value_minimum` (when specified) 
+* SHOULD NOT allow submission of zap events after `closed_at` time (when specified)
+* SHOULD NOT allow poll author to vote on their own polls
+* SHOULD hide tally results, until after a user has zapped the note
+
+## Zap vote format
+
+The zap request event (kind `9734`): 
+* MUST specify an `e` tag that references the original poll event
+  * MUST include a primary hosting relay URL in the `e` tag
+* MUST include at least 1 `p` tag specifying the recipient's key, chosen from the original poll event's list of keys 
+  * MUST include the same primary hosting relay URL in all `p` tags as is specified in the `e` tag
+* MUST include exactly 1 `poll_option` tag which references the chosen vote option by its corresponding index `n`
+* MUST include a valid `ots` tag, if the original poll event specifies a `closed_at` time
```

Comment:
Where is the `ots` tag described? I’m unable to deduce its meaning from context.

Reply:
[NIP03](03.md)

Reply:
Thanks. Recommend linking this in the text for future readers.

```
* MUST include a valid ([NIP03](./03.md)) `ots` tag, ...
```

Reply:
Added